packages/web-client/app/components/card-pay/issue-prepaid-card-workflow/index.ts

```diff
@@ -294,85 +266,36 @@ class IssuePrepaidCardWorkflow extends Workflow {
-        );
-      },
     }),
-    new WorkflowMessage({
-      author: cardbot,
+    conditionalCancelationMessage({
+      forReason: FAILURE_REASONS.UNAUTHENTICATED,
       message: 'You are no longer authenticated. Please restart the workflow.',
-      includeIf() {
-        return (
-          this.workflow?.cancelationReason === FAILURE_REASONS.UNAUTHENTICATED
-        );
-      },
     }),
-    new WorkflowMessage({
-      author: cardbot,
+    conditionalCancelationMessage({
+      forReason: FAILURE_REASONS.RESTORATION_UNAUTHENTICATED,
       message:
         'You attempted to restore an unfinished workflow, but you are no longer authenticated. Please restart the workflow.',
-      includeIf() {
-        return (
-          this.workflow?.cancelationReason ===
-          FAILURE_REASONS.RESTORATION_UNAUTHENTICATED
-        );
-      },
     }),
-    new WorkflowMessage({
-      author: cardbot,
+    conditionalCancelationMessage({
+      forReason: FAILURE_REASONS.RESTORATION_L2_ACCOUNT_CHANGED,
       message:
         'You attempted to restore an unfinished workflow, but you changed your Card Wallet address. Please restart the workflow.',
-      includeIf() {
-        return (
-          this.workflow?.cancelationReason ===
-          FAILURE_REASONS.RESTORATION_L2_ACCOUNT_CHANGED
-        );
-      },
     }),
-    new WorkflowMessage({
-      author: cardbot,
+    conditionalCancelationMessage({
+      forReason: FAILURE_REASONS.RESTORATION_L2_DISCONNECTED,
       message:
         'You attempted to restore an unfinished workflow, but your Card Wallet got disconnected. Please restart the workflow.',
-      includeIf() {
-        return (
-          this.workflow?.cancelationReason ===
-          FAILURE_REASONS.RESTORATION_L2_DISCONNECTED
-        );
-      },
     }),
-    new WorkflowMessage({
-      author: cardbot,
+    conditionalCancelationMessage({
+      forReason: FAILURE_REASONS.UNSUPPORTED_WORKFLOW_STATE_VERSION,
       message:
         'You attempted to restore an unfinished workflow, but the workflow has been upgraded by the Cardstack development team since then, so you will need to start again. Sorry about that!',
-      includeIf() {
-        return (
-          this.workflow?.cancelationReason ===
-          FAILURE_REASONS.UNSUPPORTED_WORKFLOW_STATE_VERSION
-        );
-      },
-    }),
-    new WorkflowCard({
-      author: cardbot,
-      componentName: 'workflow-thread/default-cancelation-cta',
-      includeIf() {
-        return (
-          [
-            FAILURE_REASONS.DISCONNECTED,
-            FAILURE_REASONS.ACCOUNT_CHANGED,
-            FAILURE_REASONS.UNAUTHENTICATED,
-            FAILURE_REASONS.RESTORATION_UNAUTHENTICATED,
-            FAILURE_REASONS.RESTORATION_L2_ACCOUNT_CHANGED,
-            FAILURE_REASONS.RESTORATION_L2_DISCONNECTED,
-          ] as String[]
-        ).includes(String(this.workflow?.cancelationReason));
```

Comment:
I am not sure why the prepaid card workflow uses this pattern while other workflows don't. Changing this one to use the standard check to show the default cancelation CTA did not break any tests. If anyone knows of a reason we need to preserve the logic removed here, let me know.

Reply:
Sorry I was unclear - all ctas, whether cancelation or epilogue, are not completed cards. They started that way for styling purposes, but this is something we hope to change in the issue I mentioned. 

In this case, the first cancelation CTA not being complete is going to prevent the second from being shown.

Reply:
So NextSteps boxes are not cards, they're just components. We've been treating them like action-cards which is actually not right but we never refactored it. In the case of user having insufficient funds, we only want to display the `insufficient-funds-cta` component and not the `default-cancellation-cta`.  Both of those components are using the `NextStepsBox` component underneath. It seems like because of the completion state on the next steps box (which actually should not have a completion state), everything seems to still be working.